app/code/community/Businessfactory/Roihuntereasy/Model/Cron.php

```diff
@@ -167,6 +126,157 @@ private function generateAndSaveFeed()
         return true;
     }
 
+    /**
+     * Feed generation function
+     */
+    private function generateAndSaveFeedCSV($products, $io) {
+        // CSV headers
+        $csvHeader = array(
+            'ID',
+            "Item title",
+            "Final URL",
+            "Image URL",
+            "Item description",
+            "Price",
+            "Sale price"
+        );
+
+        // write headers to CSV file
+        $io->streamWriteCsv($csvHeader);
+
+        $this->count = 0;
+        foreach ($products as $_product) {
+            switch ($_product->getTypeId()) {
+                case 'downloadable':
+                    if ($_product->getPrice() <= 0) {
+                        break;
+                    }
+//              Else same processing as simple product
+                case 'simple':
+                    $productDict = $this->get_simple_product_dict($_product);
+                    $io->streamWriteCsv($productDict);
+                    break;
+                case 'configurable':
+                    $productDictArray = $this->get_configurable_product_dict($_product);
+                    foreach ($productDictArray as $productDict){
+                        $io->streamWriteCsv($productDict);
+                    }
+                    break;
+            }
+        }
+
+    }
+
+    /**
+     * @param Mixed $_product
+     * @param XMLWriter $xmlWriter
+     */
+    private function get_configurable_product_dict($_product)
```

Comment:
You shouldn't mix method naming strategy (cameCaseStrategy and underscore_strategy). Please choose one of them.